contents/tutorials/ios-analytics.md

```diff
@@ -0,0 +1,279 @@
+  imageLight={EventsInPostHogLight} 
+  imageDark={EventsInPostHogDark} 
+  alt="Events captured in PostHog" 
+  classes="rounded"
+/>
+
+## 3. Implement the event capture code
+
+To show how to capture events with PostHog, we capture an event when the submit button on the home page is clicked. To do this, we call [PostHogSDK.shared.capture()](/docs/libraries/ios#capturing-events):
+
+```swift file=ContentView.swift
+import SwiftUI
+import PostHog
+
+// ...
+
+struct HomeView: View {
+            // ...
+
+            Button("Submit") {
+                PostHogSDK.shared.capture("home_button_clicked")
+            }
+
+            // ...
+```
+
+Refresh your app and click the button on the home page a few times. You should now see the captured event in your [PostHog activity tab](https://us.posthog.com/events).
+
+### Setting event properties
+
+When capturing events, you can optionally include additional information by setting the `properties` argument. This is helpful for breaking down or filtering events when creating [insights](/docs/product-analytics/insights).
+
+As an example, we add the value of the toggle as an event property:
+
+```swift file=ContentView.swift
+struct HomeView: View {
+            // ...
+
+            Button("Submit") {
+                PostHogSDK.shared.capture("home_button_clicked", properties: [
+                    "is_toggled_enabled": isToggleOn
+                ])
+            }
+
+            // ...
+```
+
+### Identifying users
+
+Linking events to specific users enables you to build a full picture of how they're using your product across different sessions, devices, and platforms. To link events from anonymous to specific users, we call [`PostHogSDK.shared.identify()`](/docs/libraries/ios#identify) with a `distinctId` argument. The `distinctId` must be a unique identifier for the user – usually their email or database ID.
+
+To show you an example, update the code for the login button to the following:
+
+```swift file=ContentView.swift
+struct LoginView: View {
+            // ...
+
+            Button("Login") {
+                isAuthenticated = true
+                PostHogSDK.shared.identify($email.wrappedValue)
+            }
+
+            // ...
+```
+
+Any past or future events captured after calling `identify` will now be associated with the email you provided. 
+
+To test this, press logout, fill the form in and press login. Then, in the home page, press the submit button to capture the `home_button_clicked` event. You should now see the email in the **Person** column in your [activity tab]((https://us.posthog.com/events)).
+
+<ProductScreenshot
+  imageLight={IdentifyLight} 
+  imageDark={IdentifyDark} 
+  alt="Identified events captured in PostHog" 
+  classes="rounded"
+/>
+
+Lastly, when the user logs out, you should call [`PostHogSDK.shared.reset()`](https://posthog.com/docs/libraries/ios#reset). This resets the PostHog ID and ensures that events are associated to the correct user.
+
```

Comment:
Can't get this to fully work, but might as well add it?
```suggestion
```swift file=ContentView.swift
     Button("Logout") {
                isAuthenticated = false
                PostHogSDK.shared.reset()
            }
```
```